dask/distributed/client.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, scheduler, address=None):
         self.socket = context.socket(zmq.DEALER)
         self.socket.setsockopt(zmq.IDENTITY, self.address)
         self.socket.connect(self.address_to_scheduler)
+        self.send_to_scheduler({'function': 'register'}, {})
```

Comment:
Do we want to send any identifying state to the scheduler?  The second arugument here `{}` holds metadata about the client.